app/add-equipment/page.module.css

```
@@ -10,3 +10,7 @@
   margin-bottom: 2rem;
   text-align: center;
 }
+
+:global(.dark) .title {
```

Stylelint (17.12.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)